projects/plugins/jetpack/modules/sso.php

```diff
@@ -219,14 +219,9 @@ public function render_invitation_email_message() {
 		// Enqueue the CSS for the admin create user page.
 		wp_enqueue_style( 'jetpack-sso-admin-create-user', plugins_url( 'modules/sso/jetpack-sso-admin-create-user.css', JETPACK__PLUGIN_FILE ), array(), time() );
 
-		$message = sprintf(
-			// translators: %s is a link to jetpack support site.
-			__( 'New users will receive an invite to join WordPress.com, so they can log in securely using %s', 'jetpack' ),
-			sprintf(
-				'<a class="jetpack-sso-admin-create-user-invite-message-link-sso" rel="noopener noreferrer" target="_blank" href="%s">%s</a>',
-				'https://jetpack.com/support/sso/',
-				__( 'Secure Sign On.', 'jetpack' )
-			)
+		$message = __(
```

Comment:
Could you add some extra safety with a `wp_kses` call here? The string will originally split so we wouldn't have any issues with the full HTML tag, but if we let translators customize that tag, I believe we need to ensure they only use allowed HTML attributes.

Reply:
Hi @jeherve 👋, thank you for reviewing this one, I have addressed your suggestion.